cli-plugins/socket/socket.go

```diff
@@ -92,6 +94,7 @@ func (pl *PluginServer) Addr() net.Addr {
 //
 // The error value is that of the underlying [net.Listner.Close] call.
 func (pl *PluginServer) Close() error {
+	logrus.Debug("Closing plugin server")
```

Comment:
Would you still rather I remove this line @tonistiigi? I don't really mind either way, but I expect debug logs to be quite verbose.

Reply:
Currently, we don't have many debug logs for commands, and for example in buildx where we use `--debug` for extra verbosity for build errors, this can be confusing to users who have no context of "plugin server".

Reply:
Yeah the tricky bit with the CLI (unlike the daemon, where it usually ends up in a log file) is that logging in the CLI is.. printing in the user's terminal.

Would be nice if we had a location to store logs, but .. we don't, so we should not use it except for exceptional cases.

We _could_ use "trace" level logging (one level more verbose than debug), but not sure if we pass that through currently (and not sure if containerd logs had it as option)

Reply:
Oh, that works! 
```go
logrus.Trace("Closing plugin server")
```

```
./build/docker -l trace compose version
TRAC[0000] Plugin server listening on /var/folders/zj/khjjyd1n0zjd9xc44815szxc0000gn/T/docker_cli_9e516c003cda3c20ab79b88afc18434b 
Docker Compose version v2.27.1-desktop.1
TRAC[0000] Closing plugin server                                              
```

Reply:
@tonistiigi this work for you?

Reply:
I think it should be fine; we can remove in a follow-up if it's a problem (but we need to look a bit at logging overall; I also did a quick attempt to see what it'd could look like if we switched over to containerd/log, as we did on the engine side; https://github.com/docker/cli/pull/5149